### ISSUE TITLE
Brightness control

### ISSUE THREAD

**Issue by tarasov503** [AndreyPavlenko/Fermata#448]
Everytime I mirror my phone onto my headunit, my phone's screen goes extremely dark while it stays bright on the headunit. I've tried several apps and manually changing brightness myself with no luck, it keeps going dark seconds later. Apologies if this has already been addressed, but it's driving my nuts. Please help. Using pixel 8 pro if that matters any. Thanks

**Comment by AndreyPavlenko** [comment]
Right, the brightness is set to 0 when the mirroring is started. When you touch the phone screen, it's temporary restored.

**Comment by tarasov503** [comment]
Is there a way to not have brightness at zero? The brightness is restored for literally a few seconds on my phone and then goes dark, and is extremely frustrating if I ever have to use the phone when it's connected on the headunit as well

**Comment by tarasov503** [comment]
Also,l when watching any kind of video while mirroring , it gets maybe 5-7 minutes in and then the mirror freezes the video on the head unit screen, while the video still plays on the phone ,but usually not in full screen on the phone anymore, it's like something happens that makes the video almost close/makes the video a small window on the phone and stops mirroring .

**Comment by AndreyPavlenko** [comment]
> Is there a way to not have brightness at zero?

Currently, there is no any settings for this. The brightness is restored when you touch the phone screen and, after 30 seconds of inactivity it's set to 0 again.

**Comment by AndreyPavlenko** [comment]
> Also,l when watching any kind of video while mirroring , it gets maybe 5-7 minutes in and then the mirror freezes the video on the head unit screen, while the video still plays on the phone

Do you watch the video in Fermata or some other player?

**Comment by tarasov503** [comment]
> > Also,l when watching any kind of video while mirroring , it gets maybe 5-7 minutes in and then the mirror freezes the video on the head unit screen, while the video still plays on the phone
> 
> Do you watch the video in Fermata or some other player?

It happens when I'm streaming YouTube videos on the fermata app. However, it doesn't have any problems with freezing on the headunit when watching m3u TV links on fermata. Anything longer than approx 5min on YouTube just ends up freezing when my phone screen shuts off from being inactive.. might have to find a way or special app to keep my screen on when the fermata app is open, I'm pretty confident that's the main cause because I've noticed on several occasions now of my headunit freezing the same exact time my phones display shuts off.
